### DISCUSSION TITLE
Remove NotePad++ From the Laragon Install

### DISCUSSION THREAD

**Discussion by mark2090** [leokhoa/laragon#467] [Q&A] [answered]
I already have and installed Notepad++ 32bit onto my computer.
I use the 32bit because of some plugins that don't exist in the 64bit version.

When I installed laragon, it came bundled with Notepad++ 64bit, which became a separate installation from the 32 bit and this caused me a lot of conflicts. 

At times the 32 bit opens like I want to, other times the 64 bit opens instead and that does not have the plugins I use. 
In addition, in the right-click context menu of "Open with", it lists two(2) different Notepad++ entries, but they are not distinguishable as to which version. 

**Questions**

1. What I am asking is how can I install Laragon without its bundled version of Notepad++ or remove it from the installation?
2. Also where in this forum do I post a request to the developer to not include Notepad++ or give the option to not install it with Laragon?

Thanks.

**Comment by arun07as** [comment, marked as answer]
I removed the notepadd++ by deleting its folder in `path-to-laragon/bin` folder and it worked, now I no longer see open with notepadd++ in the right-click context menu.

You will then need to change the default editor used by Laragon or you won't be able to open files such as `php.ini` directly from Laragon. To change the default editor, edit `path-to-laragon/usr/laragon.ini` and add the path to the executable of your preferred editor. For me, I prefer vscode, so I added this

```
[preferences]
Editor="C:\Users\WAC\AppData\Local\Programs\Microsoft VS Code\Code.exe"
```
**Disclaimer**
Btw, I'm not sure if deleting the folder is the proper way to do it, or whether this will leave residue files in other places. So if you can find some other way to resolve your issue it might be better to prefer that. Just leaving this here in case someone wants to just get it done quickly.

**Reply by Pen-y-Fan** [reply]
Nice 👍

The manually installed Notepad++ can be configured in place of Laragon's Notepad++:

```ini
[preferences]
Editor="C:\Program Files\Notepad++\Notepad++.exe"
```

I use the manually installed Notepad++ too, with plugins.

**Comment by mark2090** [comment]
Sorry for the delayed response.

Thanks for the info.

**Comment by skylennard** [comment]
This worked for me to remove Notepad++ (which I don't like or use) and put VSCode as my editor. Now I see the option in the menu and any text editing opens automatically in my VSCode window. Rad! Thanks!
